src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/run_conformance.py

```diff
@@ -0,0 +1,222 @@
+            f'source .env3/bin/deactivate'
+        process = Popen(command, shell=True)
+        out, err = process.communicate()
+        if err is None:
+            for line in str(out).split('\n'):
+                logger.info(line)
+        else:
+            logger.error(err)
+            raise Exception("Process failed on step: 'Model conversion'")
+        logger.info(f"Model conversion is successful. Converted models are saved to {converted_model_path}")
+        return converted_model_path
+
+    def download_and_convert_models(self):
+        logger.info("Starting model downloading and conversion")
+        self._omz_path = self.__download_repo(OMZ_REPO_URL, OMZ_REPO_BRANCH)
+        self._model_path = self._convert_models()
+        logger.info("Model downloading and conversion is finished successful")
+
+    def dump_subgraph(self):
+        subgraph_dumper_path = os.path.join(self._ov_bin_path, SUBGRAPH_DUMPER_BIN_NAME)
+        conformance_ir_path = os.path.join(self._working_dir, "conformance_ir")
+        if not os.path.exists(conformance_ir_path):
+            os.mkdir(conformance_ir_path)
+        logger.info(f"Stating model dumping from {self._model_path}")
+        cmd = f'{subgraph_dumper_path} --input_folders={self._model_path} --output_folder={conformance_ir_path}'
+        process = Popen(cmd, shell=True)
+        out, err = process.communicate()
+        if err is None:
+            for line in str(out).split('\n'):
+                logger.info(line)
+            logger.info(f"Conformance IRs were saved to {conformance_ir_path}")
+        else:
+            logger.error(err)
+            raise Exception("Process failed on step: 'Subgraph dumping'")
+        self._model_path = conformance_ir_path
+
+    def _prepare_filelist(self):
+        xmls = Path(self._model_path).rglob("*.xml")
+        filelist_path = os.path.join(self._model_path, "conformance_ir.lst")
+        with open(filelist_path, 'w') as file:
+            for xml in xmls:
+                file.write(str(xml) + '\n')
+            file.close()
+        return filelist_path
+
+    def run_conformance(self):
+        gtest_parallel_path = os.path.join(self.__download_repo(GTEST_PARALLEL_URL, GTEST_PARALLEL_BRANCH), "thirdparty", "gtest-parallel", "gtest_parallel.py")
+        worker_num = os.cpu_count()
+        conformance_path = None
+        if self._type == "OP":
+            conformance_path = os.path.join(self._ov_bin_path, OP_CONFORMANCE_BIN_NAME)
+        else:
+            conformance_path = os.path.join(self._ov_bin_path, API_CONFORMANCE_BIN_NAME)
+
+        logs_dir = os.path.join(self._working_dir, f'{self._device}_logs')
+        report_dir = os.path.join(self._working_dir, 'report')
+        parallel_report_dir = os.path.join(report_dir, 'parallel')
+        conformance_filelist_path = self._prepare_filelist()
+        if not os.path.exists(report_dir):
```

Comment:
But if it is exists, should we rewrite it silently?

Reply:
Fixed